packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -652,6 +686,101 @@ class Auth {
         return token
     }
 
+    /**
+     * TODO: TAOB Eventually this will be a wrapper method for commerce-sdk-isomorphic helper: authorizeTrustedAgent.
+     *
+     */
+    async authorizeTrustedAgent(credentials: {loginId?: string}) {
+        const slasClient = this.client
+        const codeVerifier = helpers.createCodeVerifier()
+        const codeChallenge = await helpers.generateCodeChallenge(codeVerifier)
+        const organizationId = slasClient.clientConfig.parameters.organizationId
+        const clientId = slasClient.clientConfig.parameters.clientId
+        const siteId = slasClient.clientConfig.parameters.siteId
+        const loginId = credentials.loginId || 'guest'
+        const isGuest = loginId === 'guest'
+        const idpOrigin = isGuest ? 'slas' : 'ecom'
+
+        const url = `${slasClient.clientConfig.proxy}/shopper/auth/v1/organizations/${organizationId}/oauth2/trusted-agent/authorize?${[
+            ...[
```

Comment:
Looking at this more closely, not sure if `authorizeTrustedAgent` needs to be a helper function in the `isomorphic-sdk`. Took a peek at the SLAS docs, does this endpoint match up to `getTrustedAgentAuthorizationToken` endpoint? Is there a reason why we can't call this directly from the SDK and perform that logic here instead of passing on the request URL?

Reply:
@joeluong-sfcc this is the same issue that @yunakim714 had in her Social Login work.  Basically handling the 303 See Other response is impossible inside a fetch that is run in the browser.

Reply:
Ah ok I see, thanks for clarifying. In that case if this logic could be more widely used at the SDK level we should probably expose it as a helper in the `isomorphic-sdk`

Reply:
🙏 Can ya (@bfeister / @joeluong-sfcc ) create a ticket to implement the helpers in the isomorphic sdk and replace it in the auth class? 

please include the specific helpers names in the ticket so it's clear to whoever works on it 🙏

Reply:
Done, created ticket W-16994146